23/README.md

```diff
@@ -0,0 +1,283 @@
+
+Those metrics reflect the different states of a valid transaction: accepted and finalized.
+
+Even if the reference metric is how many transactions the network finalizes per second, during this analysis we dig deeper and analyze the distribution on accepted transactions, committed etc. We want to make sure that the system is responsive most of the time, and we don't want to get fooled by an average that hides a lot of data behind it. Knowing the actual distribution of how much time it takes for a transaction to be accepted and then committed allows us to have a clear understanding of the responsiveness of the system, and this is something we won't know if we look only at the average. For this reason we developed our own tool, `bigchaindb-benchmark`.
+[BigchainDB Benchmark][bdb:benchmark-tool] is the tool we developed to generate the workload and collect statistics. It can simulate a large amount of workload by spawning multiple processes, and output a CSV file to timestamp the different steps in the life cycle of a transaction. This allows us to measure the distribution of how long it takes for a transaction to be accepted and later finalized.
+We also run other tests using a customized version of [`tm-bench`][tm:tm-bench].
+
+Another thing to consider is that BigchainDB is a multi-asset blockchain. With BigchainDB, users can create their own custom assets. Each new asset is identified by a unique value, that is the `id` of the `CREATE` transaction that minted it. Assets have their own unique history or—in more precise terms—directed acyclic graph. In order to check if a transaction is a double spend, we can safely ignore all transactions related to other assets. This means that when BigchainDB has to validate a block, it can parallelize validation, exploiting all cores in the current machine (more details to come). To understand if we could get a performance boost out of this, we integrated a new experimental feature that enables parallel validation of transactions. This feature is a proof of concept to show that higher throughput is realizable. Note: this feature skips also *Check Tx*, used by Tendermint to check a transaction before storing it in the mempool. BigchainDB validates transactions as soon as they are posted to the HTTP API, so *Check Tx* in this case is redundant. We must mention that *Check Tx* is used also to validate transactions coming from the other nodes of the network, so disabling it might allow byzantine actors to push invalid transactions to the mempool (handling invalid transactions is [actively discussed in the Tendermint issue tracker][tm:invalid-txs]). Even if an invalid transaction makes it to the mempool, it will be always discarded during the *Deliver Tx* phase, so this experimental feature will still maintain a correct state of the application. Parallel validation is an optimistic approach that will be eventually developed for specific production use cases.
```

Comment:
`Note: this feature skips also *Check Tx*,`

should say

`Note: this feature also skips *Check Tx*,`

and I wonder if we should write it CheckTx like how Tendermint does.